redact-wasm/workerd.ts

```diff
@@ -0,0 +1,54 @@
+import * as core from "./wasm/arcjet_analyze_bindings_redact.component.js";
+import type {
+  ImportObject,
+  RedactedSensitiveInfoEntity,
+  RedactSensitiveInfoConfig,
+  SensitiveInfoEntity,
+} from "./wasm/arcjet_analyze_bindings_redact.component.js";
+import componentCoreWasm from "./wasm/arcjet_analyze_bindings_redact.component.core.wasm";
```

Comment:
```suggestion

import componentCoreWasm from "./wasm/arcjet_analyze_bindings_redact.component.core.wasm";
```
Nit: I'd like to keep a space between the "normal" imports and the wasm module imports